frontend/console/src/features/timeline/TimelineEventList.tsx

```diff
@@ -0,0 +1,88 @@
+import type { Event } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { formatTimestampShort } from '../../utils'
+import { deploymentTextColor } from '../deployments/deployment.utils'
+import { TimelineCall } from './TimelineCall'
+import { TimelineDeploymentCreated } from './TimelineDeploymentCreated'
+import { TimelineDeploymentUpdated } from './TimelineDeploymentUpdated'
+import { TimelineIcon } from './TimelineIcon'
+import { TimelineIngress } from './TimelineIngress'
+import { TimelineLog } from './TimelineLog'
+
+interface EventTimelineProps {
+  events: Event[]
+  selectedEventId?: bigint
+  handleEntryClicked: (entry: Event) => void
+}
+
+const deploymentKey = (event: Event) => {
+  switch (event.entry?.case) {
+    case 'call':
```

Comment:
Not all are verb-to-verb` For example using `ftl call` will just call the verb directly. Where as `echo` might call `time` (as a verb-to-verb). We now also have ingress-to-verb as well :)

Reply:
That makes sense, mostly I'm just wondering how users would interact with this filter. For example, I might instinctively select `call` and expect ingress calls to show up in the filtered events list as well, since those are still calls, technically. Do you think we should add a layer of nesting to the checkboxes so we have calls, and then within calls, there are options for non-ingress calls vs ingress calls? And have non-ingress calls map to `case === 'call'`, ingress calls map to `case === 'ingress'`, and all calls maps to `['call', 'ingress'].includes(case)`?

Reply:
gotcha. I think that's part of the weirdness of this endpoint currently being used for the event/timeline filter panel, and this case where we want all events for a given module. So the filter panel is somewhat straight forward, but will need to be updated as well with all these new event types. :) 

This version sort of wants to know any event that came into a module so we can for example, show `calls` into `echo.echo` if they originated from an `ingress` like `echo.listEchos`. This will all def need some thought and design moving forward. We should huddle :)